package.json

```diff
@@ -56,7 +56,7 @@
     "version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
     "prepublishOnly": "grunt build",
     "postinstall": "node build/utils/postinstall.js",
-    "release": "standard-version && node build/sri-update"
```

Comment:
Maybe make `grunt build` a prerelease step instead?

Reply:
There isn't an npm `prerelease` script, so we'd have to make one...but it would only have one thing in it. We do also call `grunt build` on `prepublishOnly` so we don't publish without building first. But honestly I don't see what a custom script would gain us since this is such a simple change.

https://docs.npmjs.com/misc/scripts

Reply:
If you add a `pre` before a custom script, npm will respect it.

With a package.json that looks like:

```json
  "scripts": {
    "prerelease": "echo prerelease",
    "release": "echo release"
  },
```

Running `npm run release` will output both prerelease and release:

```
∴ npm run release

> foo@1.0.0 prerelease /Users/stephen/foo
> echo prerelease

prerelease

> foo@1.0.0 release /Users/stephen/foo
> echo release

release
```

Reply:
I'm still not really seeing what this will gain us. It also doesn't address `prepublishOnly`, which isn't the same context as `release`